### PR TITLE
Integrate bulk

### PR DESCRIPTION
**Pull request by jkurz**
* Adding enableMonitoring on create
* removing harborAPI as a dependency and integrating directly with the bulk API

### CODE REVIEW

File: cmd/harborAPI.go [hunk 1]
```diff
@@ -467,17 +467,17 @@ func UpdateContainerImage(username string, token string, shipment string, compos
...
 	shipment.Username = username
 	shipment.Token = token
 
 	//POST /api/v1/shipments
-	res, body, err := create(username, token, config.HarborURI+"/api/v1/shipments", shipment)
+	res, body, err := create(username, token, config.ShipitURI+"/v1/bulk/shipments", shipment)
 
-	if err != nil || res.StatusCode != http.StatusOK {
-		fmt.Printf("creating shipment was not successful: %v \n", body)
+	if err != nil || res.StatusCode != 201 {
```
Comment (jritsema) [inline cmd/harborAPI.go]: prefer http.StatusCreated here

File: cmd/types.go [hunk 2]
```diff
@@ -49,18 +49,22 @@ type DockerComposeService struct {
...
-	ParentShipment struct {
-		Name    string          `json:"name,omitempty"`
-		EnvVars []EnvVarPayload `json:"envVars,omitempty"`
-		Group   string          `json:"group,omitempty"`
-	}
-	BuildToken       string `json:"buildToken,omitempty"`
-	EnableMonitoring bool   `json:"enableMonitoring,omitempty"`
+	Username         string             `json:"username"`
+	Token            string             `json:"token"`
+	Name             string             `json:"name,omitempty"`
```
Comment (jritsema) [inline cmd/types.go]: Seems like we should remove `omitempty` for non-optional members?

File: cmd/up.go [hunk 3]
```diff
@@ -208,38 +208,28 @@ func getPrimaryPort(ports []PortPayload) PortPayload {
...
-	//create environment
-	newShipment.Environment = NewEnvironment{
-		Name: shipment.Env,
-		Vars: make([]EnvVarPayload, 0),
-	}
-
-	//add environment-level env vars
-	for name, value := range shipment.Environment {
-		newShipment.Environment.Vars = append(newShipment.Environment.Vars, envVar(name, value))
-	}
```
Comment (jritsema) [inline cmd/up.go]: What about env-level envvars? I think we still need those for log shipping.

**Review by jritsema — CHANGES_REQUESTED** [review]

File: cmd/types.go (hunk 2 above)
Comment (jkurz) [reply cmd/types.go]: didn't want to change it how it already was. You think we should remove?

File: cmd/up.go (hunk 3 above)
Comment (jkurz) [reply cmd/up.go]: true. I didn't think we added any environment level envVars. I'll add this back.

File: cmd/harborAPI.go (hunk 1 above)
Comment (jkurz) [reply cmd/harborAPI.go]: ok adding.

**Review by jritsema — APPROVED** [review]

**Pull request merged**